metis-core/metis-core-rest/src/main/java/eu/europeana/metis/core/rest/exception/RestResponseExceptionHandler.java

```diff
@@ -51,7 +52,8 @@
       WorkflowExecutionAlreadyExistsException.class,
       ScheduledWorkflowAlreadyExistsException.class,
       NoWorkflowExecutionFoundException.class, ExecutionException.class,
-      InterruptedException.class, UserUnauthorizedException.class, PluginExecutionNotAllowed.class})
+      InterruptedException.class, UserUnauthorizedException.class, PluginExecutionNotAllowed.class,
+      ExternalTaskException.class})
```

Comment:
I don't think that's possible because if you check the code inside the method it checks the annotation inside the exception ResponseStatus which is specific for each class. Ok I mighttry a if I can inherit them from a superclass and see if that works or not.